pkg/crypto/rings/cache_test.go

```diff
@@ -41,23 +45,32 @@ func newAccount(curve string) account {
 }
 
 func TestRingCache_BuildRing_Uncached(t *testing.T) {
-	rc := createRingCache(t)
+	// Create and start the ring cache
+	ctx, cancelCtx := context.WithCancel(context.Background())
+	defer cancelCtx()
+	rc, _ := createRingCache(ctx, t, "")
+	rc.Start(ctx)
+	defer rc.Stop()
```

Comment:
OPTIONAL: I don't actually know what the better practice is for tets, but I'd consider using a `t.Clean({/* stop here */}` approach.

Not that the current approach also works so feel free to ignore.